src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (num) => {
+    const resultSum = sum;
+
+    if (num !== undefined) {
```

Comment:
You can inverse this condition so as to not use `else`

Reply:
> You can inverse this condition so as to not use `else`

Good point, avoid else statement if you can